Client/Frontend/Browser/Onboarding/Welcome/WelcomeViewCallout.swift

```diff
@@ -0,0 +1,478 @@
+                $0.top.equalToSuperview()
+                $0.width.equalTo(20.0)
+                $0.height.equalTo(13.0)
+            }
+            
+            contentView.snp.makeConstraints {
+                $0.top.equalTo(arrowView.snp.bottom)
+                if UIDevice.isIpad {
+                    $0.width.lessThanOrEqualTo(450)
+                    $0.centerX.equalToSuperview()
+                } else {
+                    $0.leading.trailing.equalToSuperview().inset(DesignUX.padding)
+                }
+                $0.bottom.equalToSuperview()
+            }
+        } else {
+            addSubview(backgroundView)
+            addSubview(contentView)
+            addSubview(arrowView)
+            arrowView.transform = CGAffineTransform.identity.rotated(by: .pi)
+            
+            contentView.snp.makeConstraints {
+                if UIDevice.isIpad {
+                    $0.width.lessThanOrEqualTo(450)
+                    $0.centerX.equalToSuperview()
+                } else {
+                    $0.leading.trailing.equalToSuperview().inset(DesignUX.padding)
+                }
+                $0.top.equalToSuperview()
+            }
+            
+            arrowView.snp.makeConstraints {
+                $0.centerX.equalToSuperview()
+                $0.top.equalTo(contentView.snp.bottom)
+                $0.bottom.equalToSuperview()
+                $0.width.equalTo(20.0)
+                $0.height.equalTo(13.0)
+            }
+        }
+        
+        backgroundView.snp.makeConstraints {
+            $0.edges.equalTo(contentView.snp.edges)
+        }
+    }
+    
+    func animateFromCopy(view: WelcomeViewCallout, duration: TimeInterval, delay: TimeInterval) {
+        let views = [backgroundView, contentView, arrowView]
+        let otherViews = [view.backgroundView, view.contentView, view.arrowView]
+        
+        for e in views.enumerated() {
+            POPBasicAnimation(propertyNamed: kPOPViewFrame)?.do {
+                $0.fromValue = e.element.frame
+                $0.toValue = otherViews[e.offset].frame
+                $0.duration = duration
+                $0.beginTime = CACurrentMediaTime() + delay
+                e.element.layer.pop_add($0, forKey: "frame")
+            }
+        }
+    }
+    
+    func setState(state: WelcomeViewCalloutState) {
+        self.state = state
+        
+        if case .ready = state {
+            doLayout(pointsUp: true)
+        }
+        
+        primaryButton.removeAction(identifiedBy: .init(rawValue: "primary.action"), for: .primaryActionTriggered)
+        secondaryButton.removeAction(identifiedBy: .init(rawValue: "secondary.action"), for: .primaryActionTriggered)
+        
+        switch state {
+        case .welcome(let title):
+            titleLabel.do {
+                $0.text = title
+                $0.textAlignment = .center
+                $0.font = .preferredFont(for: .title1, weight: .semibold)
```

Comment:
Double check that you don't need to set `adjustsFontForContentSizeCategory` to `true` for all these labels